migration/migration.go

```diff
@@ -214,9 +215,56 @@ func PopulateCommonTypes(ctx context.Context, db *gorm.DB, witr models.WorkItemT
 	if err := createSystemBug(ctx, witr); err != nil {
 		return err
 	}
+
+	// --- Updates to the common work item types----
+	// Not doing it in a singular SQL so that this code structure can be re-used
+	// there are work item type specific changes are done.
+
+	if err := updateSystemUserstory(ctx, witr); err != nil {
+		return err
+	}
+	if err := updateSystemValueProposition(ctx, witr); err != nil {
+		return err
+	}
+	if err := updateSystemFundamental(ctx, witr); err != nil {
+		return err
+	}
+	if err := updateSystemExperience(ctx, witr); err != nil {
+		return err
+	}
+	if err := updateSystemFeature(ctx, witr); err != nil {
+		return err
+	}
+	if err := updateSystemBug(ctx, witr, db); err != nil {
+		return err
+	}
 	return nil
 }
 
+func updateSystemBug(ctx context.Context, witr models.WorkItemTypeRepository, db *gorm.DB) error {
```

Comment:
Could you rename the createXXX methods and merge them with the updateXXX?

createOrUpdateXXX

Currently we're defining the same WIT's twice, once for create and once for update. The idea was to only have to define them once.